features/steps/steps.go

```diff
@@ -280,7 +280,7 @@ func (c *Component) thisInstanceStartedEventIsQueued(input *godog.DocString) err
 		"event": b,
 	})
 
-	msg := kafkatest.NewMessage(b, 1)
+	msg, _ := kafkatest.NewMessage(b, 1)
```

Comment:
should really handle the new error with an `if err != nil` (even if it shouldn't raise one)

Reply:
This has now been addressed.